client/src/containers/PlaceList/PlaceList.js

```
@@ -17,7 +19,7 @@ class PlacesList extends React.Component {
                     places: data
                 });
             })
-            .catch((err) => { })    
+            .catch((err) => { })
```

never do that - don't swallow an error, remove the catch (it's better that an error crashes the app so that we fix it than happening silently)